src/mudblazor/.template.config/template.json

```diff
@@ -44,26 +38,19 @@
       "path": "MudBlazor.Template.Client/MudBlazor.Template.Client.csproj"
     }
   ],
+  "shortName": "mudblazor",
   "sourceName": "MudBlazor.Template",
   "sources": [
     {
       "source": "./",
       "target": "./",
       "exclude": [
-        ".template.config/**",
-        "CreateAllTemplatesForTests.bat"
+        ".template.config/**"
       ],
       "copyOnly": [
         "**/wwwroot/bootstrap/**"
       ],
       "modifiers": [
-
-        {
-          "condition": "(!IndividualLocalAuth)",
-          "exclude": [
-            "**/wwwroot/bootstrap/**"
```

Comment:
I bet I could completely delete the bootstrap folder if the remaining Auth pages (about 20 of them) are upgraded to MudBlazor.   This would take a few days.